client/blocks/author-selector/index.jsx

```diff
@@ -95,7 +95,7 @@ const SwitcherShell = React.createClass( {
 					{ ( this.props.fetchOptions.search || users.length > 10 ) &&
 						<Search
 							onSearch={ this._onSearch }
-							placeholder={ this.translate( 'Find Author…', { context: 'search label' } ) }
+							placeholder={ this.translate( 'Find Author...', { context: 'search label' } ) }
```

Comment:
New string `Find Author...` can be replaced with `Find Author…` which already has *27* translations